teacher.rb

```diff
@@ -0,0 +1,12 @@
+require 'person'
+
+class Teacher < Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] [Optional] Same here, If you applied the recommendation I mentioned above about the `id`, you don't need it here so please delete the `id` from lines 4 and 5 here
- [ ] [Required] Same here Please add `specialization` to the `initialize` method parameter as required [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=from%20Person.-,Constructor%20extends%20parent%27s%20constructor%20by%20adding%20%40specialization%20and%20a%20parameter%20for%20it.,-Override%20can_use_services%3F).